emulators/qemu-guest-agent/src/opnsense/service/templates/OPNsense/QemuGuestAgent/rc.conf.d

```diff
@@ -0,0 +1,15 @@
+{# Default setting is enabled, so that no GUI interaction is required. #}
+{% if not (helpers.exists('OPNsense.QemuGuestAgent.general.LogDebug')) or OPNsense.QemuGuestAgent.general.Enabled|default("0") != "0" %}
+{%   set optional_flags = [] %}
+{%   do optional_flags.append('-d -l /var/log/qemu-ga.log') %}
+{%   if helpers.exists('OPNsense.QemuGuestAgent.general.LogDebug') and OPNsense.QemuGuestAgent.general.LogDebug|default("0") == "1" %}
```

Comment:
we do have a shorthand for this nowadays, see the example below:

```
{%    if not helpers.empty('OPNsense.QemuGuestAgent.general.LogDebug') %}
```

Reply:
Good to know, thanks!